src/webpack/plugin/siteLoaderUtils.js

```diff
@@ -22,6 +25,21 @@ const getMdxLoader = (publicPath) => ({
         }
         return url.href;
       }],
+      [rehypeUrlInspector, {
```

Comment:
We should add this plugin based on a opt in in the config. `validateUrls: boolean`

Reply:
default to false

Reply:
Updated here https://github.com/cerner/terra-dev-site/pull/330/commits/cef108bcdeb40393bce44105c8f70ce21d06c956.